doc/recording_vcr_tests.md

```diff
@@ -1,105 +1,64 @@
-```
+The Azure CLI relies on [VCR.py](https://vcrpy.readthedocs.io/en/latest/) to record and replay HTTP communications. On top of the VCR.py, we build [ScenarioTest](scenario_base_test.md) to facilitate authoring tests.
 
-The `debug`, `run_live`, `skip_setup` and `skip_teardown` parameters in the `__init__` method are shown with their defaults and can be omitted. `debug` is the equivalent of specifying `debug=True` for all calls to `cmd` in the test (see below). Specifying `run_live=True` will cause the test to always be run with actual HTTP requests, ignoring VCR entirely. `skip_setup` and `skip_teardown` can be useful during test creation to avoid repeatedly creating and deleting resource groups.
+## Authoring Tests
 
-The `set_up` and `tear_down` methods are optional and can be omitted. For the ResourceGroupVCRTestBase these have default implementations which set up a test resource group and tear it down after the recording completes. Any commands used in these methods are only executed during a live or recorded test. These sections are skipped during playback so your test body should not rely any logic within these methods.
+https://github.com/Azure/azure-cli/blob/master/doc/scenario_base_tests.md
 
-A number of helper methods are available for structuring your script tests.
+### Tips
 
-#### cmd(command_string, checks=None, allowed_exceptions=None, debug=False)
+* Make the tests delete the Azure resources after execution.
+* Name the test method in following format: `test_<module>_<feature>`.
 
-This method executes a given command and returns the output. If the output is in JSON format, the method will return the results as a JSON object for easier manuipulation.
+## Recording Tests
 
-The `debug` parameter can be specified as `True` on a single call to `cmd` or in the init of the test class. Turning this on will print the command string, the results and, if a failure occurred, the failure.
+### Preparing
 
-The `allowed_exceptions` parameter allows you to specify one or more (as a list) exception messages that will allow the test to still pass. Exception types are not used because the CLI wraps many types of errors in a `CLIError`. There are some tests where a specific exception is intended. Add the exception message to this list to allow the test to continue successfully in the presence of this message.
+1. Set up your [development environment](configuring_your_machine.md)
+2. Log in your Azure account using `az login` command.
+3. Select the API profile you want to test. (Optional)
+4. Select the subscription you want to use for recording tests. The replaying does not rely on the knowledge of the subscription. And the credentials will be removed from the recordings.
 
-The `checks` parameter allows you to specify one or more (as a list) checks to automatically validate the output. A number of Check objects exist for this purpose. You can create your own as long as they implement the compare method (see existing checks for examples):
```

Comment:
Those are in another document https://github.com/Azure/azure-cli/blob/master/doc/scenario_base_tests.md